app.py

```diff
@@ -0,0 +1,88 @@
+import decimal
+# import os
+
+from flask import Flask, request
+from flask.json import JSONEncoder
+from flask_restful import Api, Resource
+from nerium.contrib.formatter import (AffixFormatter, CompactFormatter)
+from nerium.contrib.resultset import (SQLResultSet, TakeiResultSet)
+
+# Instantiate and configure app
+app = Flask(__name__)
+api = Api(app)
+
+
+class ResultJSONEncoder(JSONEncoder):
+    """ Add simple default encodings for Decimals and Datetimes.
+    Why `json` doesn't do this out of the box is beyond me. >:(
+    """
+
+    def default(self, obj):
+        if isinstance(obj, decimal.Decimal):
+            return str(obj)
+        elif hasattr(obj, 'isoformat'):
+            return obj.isoformat()
+        else:
+            return obj
+        return JSONEncoder.default(self, obj)
+
+
+app.config['RESTFUL_JSON'] = {'cls': ResultJSONEncoder}
```

Comment:
Curious: what does "cls" stand for here?

Reply:
@luzmcosta Class. This is essentially replacing the default Python `json` encoder (which returns 'not serializable' errors for decimals and datetimes) with our custom one.

Reply:
👍 🙇